backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -788,4 +788,26 @@ def test_request_review_but_no_review_done_since_time_of_review_request(self):
         # so I manually created a 'recruit_project' attribute.
         project_two.recruit_project = project_two
 
-        self.assertEqual(AgileCard.get_users_that_reviewed_since_last_review_request(project_two), [])
\ No newline at end of file
+        self.assertEqual(AgileCard.get_users_that_reviewed_since_last_review_request(project_two), [])
+
+
+class RepoUrlTests(TestCase):
+
+    def setUp(self):
+        self.card = factories.AgileCardFactory()
+        self.card.recruit_project.repository = git_real_factories.RepositoryFactory()
+        self.card.save()
+        self.repository = self.card.recruit_project.repository
+        self.card_no_repo = factories.AgileCardFactory(
+                            content_item=factories.ContentItemFactory(
+                            content_type=ContentItem.WORKSHOP),
+                            status=AgileCard.READY,
+                            recruit_project=None
+                            )
+
+    def test_repository_link_is_returned_and_that_it_is_an_actual_repository(self):
+        self.assertEqual(self.card.repo_url, self.repository.ssh_url)
+
+    def test_none_is_returned_for_card_not_linked_to_a_repository(self):
+        self.assertEqual(self.card_no_repo.repo_url, None)
```

Comment:
If `card_no_repo` is only used in this one function, why is it defined so far away in the code? 

If something is used only in one place, then define it in that place.